app/controllers/placements/providers/placements_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
     @pagy, @placements = pagy(placements.merge(query))
     @placements = @placements.decorate
 
+    filter_subjects_by_phase
+    filter_schools_by_phase
```

Comment:
Rather and putting this here. Consider putting this as part of the `schools_scope` method.

Reply:
I haven't been able to incorporate this without affecting the initialization of @school across the app 🤔 I thought this would work but it didn't:

```
  def schools_scope
    if filter_params[:only_partner_schools].present?
      @schools = @provider.partner_schools
    else
      @schools = Placements::School.all
    end
    
    filter_by_school_phase
  end
```

I get 58 instances of an error saying that `.order_by_name` can't act on `nil`.